content/master/concepts/packages.md

```diff
@@ -1,513 +1,489 @@
+Control the Configuration upgrade behavior with a
+{{<hover label="revision" line="6">}}revisionActivationPolicy{{</hover>}}.
 
-To push a Provider package, execute the following command:
+The {{<hover label="revision" line="6">}}revisionActivationPolicy{{</hover>}} 
+options are:
+* `Automatic` - (**default**) Automatically use the latest Configuration version
+  available in the cache. 
+* `Manual` - Require the current Configuration in use to be manually set. 
 
-```
-crossplane push provider xpkg.upbound.io/crossplane-contrib/provider-gcp:v0.22.0
+For example, to change the upgrade behavior to require manual upgrades, set 
+{{<hover label="revision" line="6">}}revisionActivationPolicy: Manual{{</hover>}}.
+
+```yaml {label="revision"}
+apiVersion: pkg.crossplane.io/v1
+kind: Configuration
+metadata:
+  name: platform-ref-aws
+spec:
+  revisionActivationPolicy: Manual
+# Removed for brevity
 ```
 
-To push a Configuration package, execute the following command:
+{{<hint "important" >}}
+Crossplane only upgrades a Configuration if a newer version is in the package 
+cache.   
+By default the Crossplane 
+[`packagePullPolicy`](#configuration-package-pull-policy) doesn't
+download new Configuration versions, even if they're available.
+{{< /hint >}}
 
-```
-crossplane push configuration xpkg.upbound.io/crossplane-contrib/my-org-infra:v0.1.0
-```
+Read the [Configuration package revision](#configuration-revisions) 
+section for more information on the use of package revisions.
 
-> Note: Both of the above commands assume a single `.xpkg` file exists in the
-> directory. If multiple exist or you would like to specify a package in a
-> different directory, you can supply the `-f` flag with the path to the
-> package.
+#### Install a Configuration from a private registry
 
-## Installing a Package
+Like Kubernetes uses `imagePullSecrets` to 
+[install images from private registries](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/), 
+Crossplane uses `packagePullSecrets` to install Configuration packages from a 
+private registry. 
 
-Packages can be installed into a Crossplane cluster using the Crossplane CLI.
+Use {{<hover label="pps" line="6">}}packagePullSecrets{{</hover>}} to provide a
+Kubernetes secret to use for authentication when downloading a Configuration 
+package. 
 
-To install a Provider package, execute the following command:
+{{<hint "important" >}}
+The Kubernetes secret must be in the same namespace as Crossplane.
+{{</hint >}}
 
-```
-crossplane install provider xpkg.upbound.io/crossplane-contrib/provider-gcp:v0.22.0
-```
+The {{<hover label="pps" line="6">}}packagePullSecrets{{</hover>}} is a list of
+secrets.
 
-To install a Configuration package, execute the following command:
+For example, to use the secret named
+{{<hover label="pps" line="6">}}example-secret{{</hover>}} configure a 
+{{<hover label="pps" line="6">}}packagePullSecrets{{</hover>}}.
```

Comment:
It is worth adding a note stating that this secret needs to be in the namespace where Crossplane runs.